types/yargs/v15/yargs-tests.ts

```diff
@@ -305,8 +305,9 @@ function Argv$command() {
             (argv) => {
                 console.dir(argv.url);
             },
-            // middlewares
-            [],
+            [(argv) => {
+                console.log(argv.url);
```

Comment:
do a type assertion on `argv.url` instead - ie `// $ExpectType string`

Reply:
Done.

Reply:
Not done. 😅 

Still working on it.

Reply:
I think we're good, this time.